introduction.md

```diff
@@ -4,3 +4,8 @@ This is a sample markdown file to do "Mastering GitHub" course of codeschool.
 
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?